packages/array-normed/types.d.ts

```diff
@@ -1,4 +1,4 @@
-declare module 'ml-array-normed' {
+declare namespace ArrayNormed {
```

Comment:
```suggestion
declare namespace mlArrayNormed {
```

Let's name all of these `mlCamelCase` so that the auto-import matches our standards